client/app/private-database/database/certificate/private-database-database-certificate.controller.js

```diff
@@ -0,0 +1,20 @@
+angular
+    .module("App")
+    .controller("PrivateDatabaseCertificateCtrl", class PrivateDatabaseCertificateCtrl {
+
+        constructor ($scope, $stateParams, Alerter, PrivateDatabase) {
+            this.$scope = $scope;
+            this.$stateParams = $stateParams;
+            this.alerter = Alerter;
+            this.privateDatabaseService = PrivateDatabase;
+        }
+
+        $onInit () {
+            this.productId = this.$stateParams.productId;
+            this.database = this.$scope.currentActionData;
+
+            this.model = {
+                certificate: _.get(this.database, "tlsCa", null)
```

Comment:
I feel like the difference between `null` and `undefined` shouldn't matter and the code should be handling `null` and `undefined` the same way, thefore `this.database.tlsCa` could suffise, exception of course if `this.database` can be `null` or `undefined`

(I know this is a complexe sentence that I may not have phrased correctly, come to me if you want to have more data ;) )